zebra-network/src/constants.rs

```diff
@@ -95,6 +95,13 @@ pub const EWMA_DEFAULT_RTT: Duration = Duration::from_secs(20 + 1);
 /// better peers when we restart the sync.
 pub const EWMA_DECAY_TIME: Duration = Duration::from_secs(200);
 
+/// OS-specific error when the port attempting to be opened is already in use.
+pub const PORT_IN_USE_ERROR: &str = if cfg!(unix) {
+    "already in use"
+} else {
+    "one usage"
```

Comment:
Use Windows error from:
https://github.com/ZcashFoundation/zebra/pull/1535/checks?check_run_id=1788192685#step:7:1553

Full panic message:
```
Opening metrics endpoint listener 127.0.0.1:49817 failed: Hyper(hyper::Error(Listen, Os { code: 10013, kind: PermissionDenied, message: "An attempt was made to access a socket in a way forbidden by its access permissions." })). Hint: Check if another zebrad or zcashd process is running. Try changing the metrics endpoint_addr in the Zebra config.
```

```suggestion
    "access a socket in a way forbidden by its access permissions"
```

Reply:
We might need to turn this into a regex so we can match both "one usage" and "access a socket in a way forbidden by its access permissions", but I want to check the exact error messages for both cases.